### PR TITLE
Support for pox calls in flow tests

### PR DESCRIPTION
**Pull request by friedger**
This PR
* adds support for any contract calls in flow tests, in particular calls to pox contract
* updates the example to use the local clarunit module

### CODE REVIEW

**Review by moodmosaic — DISMISSED** [review]
LGTM. Added some comments on possible edge case exploration.

**Comment by wileyj** [comment]
@friedger anything you think should be addressed? i'd prefer to publish the other updates along with this so nothing is dangling

File: src/parser/clarity-parser-flow-tests.ts [hunk 1]
```diff
@@ -127,22 +130,24 @@ export function extractContractCalls(lastFunctionBody: string, simnet: Simnet) {
...
  * handle (unwrap! (contract-call? ...)) statements
- * @param statement 
- * @returns 
+ * @param statement
+ * @returns
  */
 function extractUnwrapInfo(statement: string, simnet: Simnet): CallInfo | null {
   const match = statement.match(
-    /\(unwrap! \(contract-call\? \.(.+?) (.+?)(( .+?)*)\)/
+    /\(unwrap! \(contract-call\? (?:\.(.+?)|'(.+?)) (.+?)(( .+?)*)\)/
```
[the remark this answers is not in the document]
Comment (friedger) [reply src/parser/clarity-parser-flow-tests.ts:138]: We can assume that the input is valid clarity code. How would I generate that?

**Comment by friedger** [comment]
@wileyj We can improve the test coverage in the next version.

**Review by wileyj — APPROVED** [review]

**Comment by wileyj** [comment]
> @wileyj We can improve the test coverage in the next version. Let's publish it :-)

published from #17  as `0.1.0`
https://www.npmjs.com/package/@stacks/clarunit
